src/test/java/com/rakovets/course/java/core/practice/generic_types/PairTest.java

```diff
@@ -0,0 +1,27 @@
+package com.rakovets.course.java.core.practice.generic_types;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class PairTest {
+
+    @Test
+    void swap() {
+        Pair<Integer, String> pair = new Pair<>(101, "kek");
+
+        Pair<String, Integer> swappedPair = Pair.swap(pair);
+        String cheburek = swappedPair.getVar1() + swappedPair.getVar2();
+
+        Assertions.assertEquals("kek101", cheburek);
```

Comment:
Best practice:
* override method `equels` for `Pair`
OR
* test EACH field separately BUT in one test method